be/src/exec/vectorized/analytor.cpp

```diff
@@ -120,10 +121,15 @@ Status Analytor::prepare(RuntimeState* state, ObjectPool* pool, RuntimeProfile*
             if (!state->enable_pipeline_engine()) {
                 return Status::NotSupported("The NTILE window function is only supported by the pipeline engine.");
             }
-
             _need_partition_materializing = true;
         }
 
+        if (fn.name.function_name == "sum" || fn.name.function_name == "avg" || fn.name.function_name == "count") {
```

Comment:
may be we need disable sum(float) sum(double) ?

Reply:
Emm, could you explain why sum(float) should be excluded? I didn't get it.

Reply:
1.2 + 2.3 - 1.2 maybe not equals to 2.3
and when we add a small value and large maybe not have the same behaviour

Reply:
Yes, but I think the accuracy loss is acceptable for float-point type.

The loop algo and the removable cumulative algo may produce different results, and both of them are the right answer.